hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts

```diff
@@ -443,6 +443,19 @@ class HassioSnapshotDialog extends LitElement {
       return;
     }
 
+    if (window.location.href.includes("ui.nabu.casa")) {
+      const confirm = await showConfirmationDialog(this, {
+        title: "Potential slow download",
+        text:
+          "Downloading snapshots over the Nabu Casa URL will take some time, it is recomended to use your local URL instead, do you want to continue?",
+        confirmText: "continue",
+        dismissText: "cancel",
```

Comment:
can we localize this? `title`, `text`, `confirmText` and `dismissText`

Reply:
Nope